service-deployment/src/grafana/pai-nodeview-dashboard.json.template

```diff
@@ -848,14 +748,14 @@
         "allFormat": "glob",
         "allValue": null,
         "current": {
-          "text": "10.0.1.5:9100",
-          "value": "10.0.1.5:9100"
+          "text": "10.151.40.202:9100",
+          "value": "10.151.40.202:9100"
```

Comment:
What does this IPs means?

Reply:
Solved in latest commit

Reply:
Solved in latest commit

Reply:
Solved in latest commit

Reply:
@sterowang: should we hide internal ips in code?

Reply:
Sorry I forgot to modify this file. Now it is set to "" in the latest commit and works well. @sterowang @hwuu